<?xml version="1.0"?>
<launch>
     


     <arg name="dbg_args"   default="gdb -ex run --args"/>
     <arg name="debug_args"   default=""/>

     <node pkg="decent_voronoi" type="decent_voronoi_virtual_agent_ros" name="decent_voronoi_virtual_agent_ros_node" launch-prefix="$(arg debug_args)" output="screen">
         
          <param name="online_update_rate" value="10"/> <!-- unit : hz -->
          <param name="agent_num" value="10"/> <!-- unit : hz -->
          <param name="virtual_agent_start_index" value="0"/>
	  <param name="y_axis_swap" value="false"/> 
          <param name="domain_height" value="20"/> <!-- unit : meter -->
          <param name="domain_width" value="20"/> <!-- unit : meter -->
          <param name="y_axis_swap" value="false"/> 
          <rosparam param="agent_init_pose">   <!--  x_pose, y_pose, max_speed-->
           [
            0.2, 19.8, 0.4,
            0.2, 19.6, 0.4,
            0.2, 19.4, 0.4,
            0.2, 19.2, 0.4,
            0.2, 19.0, 0.4,
            0.4, 19.8, 0.8,
            0.4, 19.6, 0.8,
            0.4, 19.4, 0.8,
            0.4, 19.2, 0.8,
            0.4, 19.0, 0.8,
            8, 48, 4,
            8, 56, 4,
            8, 64, 4,
            8, 72, 4,
            8, 80, 4,
            16, 48, 4,
            16, 56, 4,
            16, 64, 4,
            16, 72, 4,
            16, 80, 4  
                  ]

          </rosparam>
     </node>

</launch>
